Cargo.toml

```diff
@@ -225,6 +225,7 @@ data-encoding = { version = "2", default-features = false }
 deadpool-postgres = { version = "0.13", default-features = false }
 dialoguer = { version = "0.10", default-features = false }
 docker_credential = { version = "1.3.1", default-features = false }
+etcetera = { version = "0.8", default-features = false }
```

Comment:
can we kill the `homedir` crate if we have this?

Reply:
Oh I was going to and forgot. Thanks for the reminder